ring_pinkie_flexion_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: ring_pinkie_flexion
    joint_speed: 5
    motor:
        id: 38
        init: 2048
        min: 0
        max: 4095